Tests/designspaceLib/data/out/.gitignore

```diff
@@ -0,0 +1,2 @@
+# This folder is meant to receive test output (DS5 -> DS4 conversions)
+*
```

Comment:
why not use a temporary folder instead of a gitignored one?

Reply:
Will do